packages/zoe/test/unitTests/contracts/test-publicSwap.js

```diff
@@ -7,12 +7,11 @@ import { setup } from '../setupBasicMints';
 
 const publicSwapRoot = `${__dirname}/../../../contracts/publicSwap`;
 
-test.skip('zoe - publicSwap', async t => {
+test('zoe - publicSwap', async t => {
   try {
     const { assays: defaultAssays, mints } = setup();
     const assays = defaultAssays.slice(0, 2);
-    const zoe = await makeZoe({ require });
-    const escrowReceiptAssay = zoe.getEscrowReceiptAssay();
+    const zoe = makeZoe({ require });
```

Comment:
OMG `require` as an endowment! Why is this safe? I just looked, and publicSwap.js never says `require`.

Reply:
I think whether this is safe or not belongs in another issue/review, since there are no changes related to this in this PR. My understanding is that the `require` endowment is so that the contract source can be bundled, but we should look into better ways of doing it.